### PR TITLE
chore(deps): upgrade dependencies

### PR DESCRIPTION
**Pull request by Ja7ad**
## Description

Upgraded dependencies

### CODE REVIEW

**Review by kehiy — APPROVED** [review]
LGTM. based on documents, will be good if you check workflows and socker file dependencies and if they had any updates.

see: #1202

File: .github/workflows/releaser.yml [hunk 1]
```diff
@@ -267,4 +267,4 @@ jobs:
       with:
         files: .github/releasers/pactus_downloader.sh
       env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (b00f) [inline .github/workflows/releaser.yml:270]: Ensure there is always a new line at the end of each file. Please install EditorConfig from https://editorconfig.org/.

File: www/grpc/gen/docs/json-rpc.md [hunk 2]
```diff
@@ -243,7 +243,7 @@ parameters.</p>
   </tr>
   <tr>
     <td class="fw-bold">verbosity</td>
-    <td> string</td>
+    <td> numeric</td>
```
Comment (b00f) [inline www/grpc/gen/docs/json-rpc.md:246]: Why?

**Review by b00f — CHANGES_REQUESTED** [review]

File: util/testsuite/testsuite.go [hunk 3]
```diff
@@ -163,10 +163,10 @@ func (ts *TestSuite) RandAmount() amount.Amount {
 }
 
 // RandAmountRange returns a random amount between [min, max).
-func (ts *TestSuite) RandAmountRange(min, max amount.Amount) amount.Amount {
-	amt := amount.Amount(ts.RandInt64NonZero(int64(max - min)))
+func (ts *TestSuite) RandAmountRange(minVal, maxVal amount.Amount) amount.Amount {
```
Comment (b00f) [inline util/testsuite/testsuite.go:166]:
```suggestion
func (ts *TestSuite) RandAmountRange(minAmt, maxAmt amount.Amount) amount.Amount {
```

File: util/encoding/encoding_test.go [hunk 4]
```diff
@@ -264,7 +264,7 @@ func TestVarBytesEncodingErrors(t *testing.T) {
 	tests := []struct {
 		in       []byte // Byte Array to write
 		buf      []byte // Encoding bytes
-		max      int    // Max size of fixed buffer to induce errors
+		maxVal   int    // Max size of fixed buffer to induce errors
```
Comment (b00f) [inline util/encoding/encoding_test.go:267]:
```suggestion
		maxSize   int    // Max size of fixed buffer to induce errors
```

File: util/encoding/encoding_test.go [hunk 5]
```diff
@@ -92,7 +92,7 @@ func TestElementEncoding(t *testing.T) {
 func TestElementEncodingErrors(t *testing.T) {
 	tests := []struct {
 		in       any   // Value to encode
-		max      int   // Max size of fixed buffer to induce errors
+		maxVal   int   // Max size of fixed buffer to induce errors
```
Comment (b00f) [inline util/encoding/encoding_test.go:95]:
```suggestion
		maxSize   int   // Max size of fixed buffer to induce errors
```

File: sortition/vrf_test.go [hunk 6]
```diff
@@ -20,12 +20,12 @@ func TestVRF(t *testing.T) {
 		seed := ts.RandSeed()
 		t.Logf("seed is: %x \n", seed)
 
-		max := uint64(1 * 1e6)
-		index, proof := sortition.Evaluate(seed, valKey.PrivateKey(), max)
+		maxSize := uint64(1 * 1e6)
```
Comment (b00f) [inline sortition/vrf_test.go:23]: revert this file also

**Review by b00f — APPROVED** [review]

**Pull request merged**